modules/installation-disk-partitioning-upi-templates.adoc

```diff
@@ -35,6 +35,54 @@ Because `/var` must be in place before a fresh installation of {op-system-first}
 If you follow the steps to create a separate `/var` partition in this procedure, it is not necessary to create the Kubernetes manifest and Ignition config files again as described later in this section.
 ====
 
+.Prerequisites
+
+* If container storage is on the root partition:
+** Then, ensure that this root partition is mounted with the "pquota" option by including the `rootflags=pquota` option in the grub command line.
```

Comment:
s/"pquota"/`pquota`

s/grub/GRUB (assuming it's the GRUB I'm familiar with)

To avoid some repetition, could also:
"mounted with the `pquota` option by including ~the~ `rootflags=pquota` ~option~ in the GRUB command line."